src/App.tsx

```diff
@@ -25,10 +26,11 @@ export default function App() {
       authorizationUrl: "", // /projects/{projectID}/oauth2/{providerId}/authorizationUrl
     },
     error: {
-      errorFunction: (msg: string) => {
-        alert(msg)
+      errorFunction: (msg: string, response?: AxiosResponse) => {
+        alert(msg);
+        console.log(response)
       },
-      errorMethod: "default",
+      errorMethod: "customFunction",
```

Comment:
Why do you want customFunction should be the default errorMethod?

Reply:
![image](https://github.com/wavemaker/rest-client-ui/assets/115954698/ed90995f-c253-45fa-ba3e-f27f98334320)
customFunction is not the default method. It is one of the methods.

Reply:
Okay